CHANGELOG.md

```diff
@@ -11,6 +11,22 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 0.9.3 - 2020-09-03
+
+### Changed
+
+- Rename *internal* API group from "networking.antrea.tanzu.vmware.com" to "controlplane.antrea.tanzu.vmware.com". ([#1147](https://github.com/vmware-tanzu/antrea/pull/1147), [@jianjuns])
+   * This API is served by the Antrea Controller and consummed by Agents (directly) and antctl (through the K8s apiserver using an APIService)
+   * Antrea Controller deletes the previous APIService on startup to avoid issues (e.g. with namespace deletion)
+   * Users should not be affected unless they consummed the API directly which is unlikely; during upgrade from a previous version, NetworkPolicy enforcement may be disrupted until the upgrade is complete
```

Comment:
Could we make it clearer here - it is NetworkPolicy changes might not be realized and new Pods might not have NetworkPolicies applied, until Controller and all Agents upgrade to the new version.

Reply:
Done. In the future it would be good to have detailed information included in the PR description so that I can link to that in the release notes.